### ISSUE TITLE
Google App engine support

### ISSUE THREAD

**Issue by shayanb** [blocktrail/blocktrail-sdk-python#7]
Has anyone tried to port this sdk to Google App engine?  is there any plans to support app engine?

**Comment by shayanb** [comment]
I put together a fork that works on Appengine: https://github.com/shayanb/blocktrail-sdk-python

**Comment by rubensayshi** [comment]
hmm, I thought the `requests` package would work on GAE, maybe I should switch the sdk to https://pypi.python.org/pypi/urllib3 instead, which does claim to support GAE

**Comment by shayanb** [comment]
yeah it might be a bit tricky to have a universal library that works on all platforms, for the fork I used `from google.appengine.api import urlfetch` that is just specific to GAE, but I've used urllib2 before for the same issue and it worked fine, urlib3 could work too.

```
try:
    from urllib2 import urlopen
except ImportError:
    from urllib.request import urlopen
```

**Comment by shayanb** [comment]
using `client.transaction(tx)` , it sometimes ends up with this error on GAE:

> '_URLFetchResult' object has no attribute 'reason' : {}

Any ideas?

p.s I'm using `from google.appengine.api import urlfetch`

**Comment by shayanb** [comment]
@rubensayshi they are the double-spends (malleability) tx-hashes that fail to reply.

**Comment by rubensayshi** [comment]
can you give me one of the failing TXs? if you don't want to publicly link one then mail me ruben@blocktrail.com

**Comment by peterlie** [comment]
requests-toolbelt is helpful to get requests working on App Engine: https://github.com/sigmavirus24/requests-toolbelt
